community-tutorials/ubuntu-docker-install/01-en.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Install Docker-CE on Ubuntu 20.04
+description: How to setup Docker-CE on Ubuntu 20.04
+updated_at: 2021-11-04
+slug: ubuntu-docker-install
+author_name: Marcel Aust
+author_url: https://marcel-aust.de
+author_image: -
+author_bio: -
+tags: [shell,docker,ubuntu,portainer,docker-compose] 
+netcup_product_url: https://www.netcup.de/bestellen/produkt.php?produkt=2000
+language: en
+available_languages: en
+---
+
+# Introduction
+This tutorial explains how to install Docker-CE on Ubuntu 20.04.
+With this tutorial you will learn how to install and configure Docker on Ubuntu to use it with many different apps and services.
+In addition this tutorial describes how to use Docker with **ufw** and additionally install and setup **docker-compose** and **portainer**.
+You should have a basic understanding of linux and shell commands. A fresh install of Ubuntu 20.04 (may work with other versions too) is required.
+
+This tutorial is based on the [offical tutorial provided by docker](https://docs.docker.com/engine/install/ubuntu/) with a few extra notes and additions.
+
+# Requirements
+Docker can be used on nearly any Root- or V-Server. Netcups server's architecture is amd64. You need to be root and/or have sudo privileges to install packages.
+
+# Step 1 - (Preparation)
+We will install Docker by using Ubuntus APT package manager. To fetch the latest package database, use:
+```bash
+    sudo apt-get update
+```
+
+To add a custom package repository for docker we need a few apt-tools which might not be already present. Use the following command to install them:
+
+```bash
+sudo apt-get install \
```

Comment:
I had the problem with a fresh installation that my server could not resolve the hostname of **de.archive.ubuntu.com** with this command and therefore the packages were not installed.

I solved this by adding the hostname to the **/etc/hosts** file by myself:

`nano /etc/hosts`
`141.76.2.3 de.archive.ubuntu.com`

Could you please add this accordingly?